CHANGELOG.md

```diff
@@ -1,10 +1,9 @@
 # Yii Test Support Change Log
 
 
-## 1.2.1 under development
-
-- no changes in this release.
+## 1.2.1 March 05, 2021
```

Comment:
```suggestion
## 1.2.1 under development
```

Reply:
Before merge will be changed to actual date